<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="navigationBG" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#navigationBG {
background-color: #ccdde8;

}

</string>
    </property>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/logout.png</normaloff>../img/logout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="systemLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>15</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#systemLabel {
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: #034488;
}</string>
     </property>
     <property name="text">
      <string>Add Genre</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rect1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#rect1{
background-color: white;
border: 1px white;
border-radius: 20px;
}</string>
    </property>
    <widget class="QLineEdit" name="genreName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>7</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#genreName{
border: TRANSPARENT;
}</string>
     </property>
     <property name="placeholderText">
      <string>Genre Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addGenreButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#addGenreButton{
background-color: #ccdde8;
border: 1px white;
border-radius: 20px;
}
#bookTicketButton:hover{
background-color: #ccdde8;
border: 1px black;
border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Add Genre</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
